charts/helm-dep-installer/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+---
+{{- $configMap := include "generic-dep-installer.cm-name" . }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $configMap }}
+  annotations:
+    "helm.sh/hook-delete-policy": "hook-succeeded,hook-failed,before-hook-creation"
```

Comment:
I only removed hook-failed for k8s job resources, configmap serviceaccount and clusterrolebindings will get removed because those are static resources. Wdyt?

Reply:
works 👍